browser-test/src/support/admin_predicates.ts

```diff
@@ -1,13 +1,122 @@
 import {Page} from 'playwright'
 import {waitForPageJsLoad} from './wait'
 
+type PredicateSpec = {
+  questionName: string
+  action?: string | null
+  scalar: string
+  operator: string
+  value?: string
```

Comment:
This is unused, either add tests that use it or remove it?  Having the caller box it into a list when needed seems fine to me FWIW

Reply:
It's used by the existing tests